### ISSUE TITLE
ImportError: No module named 'webbrowser'

### ISSUE THREAD

**Issue by rourke** [yadayada/acd_cli#439]
I installed acd_cli on my Synology NAS (ARMv5). The acd_cli command returns an error with oath: `ImportError: No module named 'webbrowser'`. I executed `help('modules')` in python3 and the module webbrowser is not in there.

I managed to do a workaround. I had the oauth_data file from a VPS I authorized before. I placed that file in ~/.cache/acd_cli/, and commented out `import webbrowser` from [oauth.py](https://github.com/yadayada/acd_cli/blob/af929608f6279fca56e6c9ac6c48bd76d4ff1dcb/acdcli/api/oauth.py). After that it worked. 

@rubenvarela this is the issue we talked about.

**Comment by yadayada** [comment]
This should not happen, because the 'webbrowser' module is in Python's standard library (https://docs.python.org/3.0/library/), even in Python 2.

**Comment by rourke** [comment]
I solved my problem, and as expected it's on my end.

I installed [OPKG](https://github.com/Entware-ng/Entware-ng/wiki/Install-on-Synology-NAS), and with it I installed the python3-pip package, which obviously has a python3 dependency. It installed python3 seperately and got priority in PATH over the correct python3 installation. Since I'd still like to have pip3, I moved `/opt/bin:/opt/sbin` to the back of PATH.

Not sure if this is best practice, but I have no idea how to uninstall the incorrect version of python3 without losing python3-pip.

Anyway, issue can be closed. Thank you for your time.
